libyul/Object.h

```diff
@@ -68,7 +68,15 @@ struct Object: ObjectNode
 
 	/// @returns the set of names of data objects accessible from within the code of
 	/// this object.
-	std::set<YulString> dataNames() const;
+	std::set<YulString> qualifiedDataNames() const;
+
+	/// @returns tuple consisting of
+	/// 	bool true if _qualifiedName is referring to data
```

Comment:
In which case can it be false? In that:?
```suggestion
	/// 	bool true if _qualifiedName is referring to data (instead of code)
```

Reply:
Or a full object instead of a single data element inside an object?